core/standard/requirements/autodiscovery/REQ_autodiscovery_searchable_link.adoc

```diff
@@ -0,0 +1,8 @@
+[[req_autodiscovery_searchable-catalog-links]]
+[width="90%",cols="2,6a"]
+|===
+^|*Requirement {counter:req-id}* |*/req/autodiscovery/searchable-catalog-links*
+
+^|A |A web page SHALL include one or more links (`rel="http://www.opengis.net/def/rel/ogc/1.0/ogc-catalog"`) that point to <<clause-searchable-catalog,searchable catalogs>> associated with the page.
```

Comment:
Suggest `http://www.opengis.net/def/rel/ogc/1.0/ogc-catalog-searchable`

Reply:
@tomkralidis I left it as "catalog" as sort of a default value because I anticipate. that  most catalogs would be searchable catalogs.  However, I suppose for consistency sake with the rest of the rel value we'll go with "ogc-catalog-searchable".